### DISCUSSION TITLE
The file exlporer covers the taskbar

### DISCUSSION THREAD

**Discussion by kristal440** [valinet/ExplorerPatcher#3101] [General]
I have win 11 and the win 10 taskbar set with EP. Sometimes the file explorer just covers up the taskbar, I don't know why it happens and I cannot reproduce it to show.

**Comment by pyrates999** [comment]
Is automatically hide the taskbar set to on?

**Comment by pyrates999** [comment]
If you have classic shell installed, uninstall it as it has been replaced by open shell.

If you're running an insider build, this issue comes back every once in a while it seems.  This isn't caused by EP.

duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1759

please close this.
